Troubleshooting/heal-info-and-split-brain-resolution.md

```diff
@@ -0,0 +1,461 @@
+### Example
+Consider a replica volume "test" with 2 bricks b1 and b2;
+self-heal daemon off, mounted at /mnt.
+
+`gluster volume heal test info`
+~~~
+Brick \<hostname:brickpath-b1>  
+<gfid:aaca219f-0e25-4576-8689-3bfd93ca70c2> - Is in split-brain
+<gfid:39f301ae-4038-48c2-a889-7dac143e82dd> - Is in split-brain
+<gfid:c3c94de2-232d-4083-b534-5da17fc476ac> - Is in split-brain
+<gfid:6dc78b20-7eb6-49a3-8edb-087b90142246> 
+
+Number of entries: 4
+
+Brick <hostname:brickpath-b2>
+/dir/file2 
+/dir/file1 - Is in split-brain
+/dir - Is in split-brain
+/dir/file3 
+/file4 - Is in split-brain
+/dir/a 
+
+
+Number of entries: 6
+~~~
+
+### Analysis of the output
+It can be seen that  
+A) from brick b1, 4 entries need healing:   
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1) file with gfid:6dc78b20-7eb6-49a3-8edb-087b90142246 needs healing  
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2) "aaca219f-0e25-4576-8689-3bfd93ca70c2",
+"39f301ae-4038-48c2-a889-7dac143e82dd" and "c3c94de2-232d-4083-b534-5da17fc476ac"
+ are in split-brain
+
+B) from brick b2 6 entries need healing-  
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1) "a", "file2" and "file3" need healing  
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2) "file1", "file4" & "/dir" are in split-brain  
+
+# 2. Volume heal info split-brain
+Usage: `gluster volume heal <VOLNAME> info split-brain`  
+This command only shows the list of files that are in split-brain. The output is therefore a subset of `gluster volume heal <VOLNAME> info`
+
+### Example
+`gluster volume heal test info split-brain`
+~~~
+Brick <hostname:brickpath-b1>
+<gfid:aaca219f-0e25-4576-8689-3bfd93ca70c2>
+<gfid:39f301ae-4038-48c2-a889-7dac143e82dd>
+<gfid:c3c94de2-232d-4083-b534-5da17fc476ac>
+Number of entries in split-brain: 3
+
+Brick <hostname:brickpath-b2>
+/dir/file1
+/dir
+/file4
+Number of entries in split-brain: 3
+~~~
+Note that similar to the heal info command, for gfid split-brains (same filename but different gfid) 
+their parent directories are listed to be in split-brain.
+
+# 3. Resolution of split-brain using gluster CLI
+Once the files in split-brain are identified, their resolution can be done
+from the gluster command line using various policies. Entry/gfid split-brain resolution is not supported via this method. Split-brain resolution commands let the user resolve data and metadata split-brain using the following policies:
+
+## i) Select the bigger-file as source
+This command is useful for per file healing where it is known/decided that the
+file with bigger size is to be considered as source.   
+`gluster volume heal <VOLNAME> split-brain bigger-file <FILE>`  
+Here, `<FILE>` can be either the full file name as seen from the root of the volume
+(or) the gfid-string representation of the file, which sometimes gets displayed
+in the heal info command's output. Once this command is executed, the replica containing the <FILE> with bigger
+size is found out and heal is completed with that bricks as a source.
```

Comment:
nit: s/bricks/brick

Also the word `<FILE>` is not rendered unless you put them between back ticks.